### PR TITLE
Right margin wrapping

### PR DESCRIPTION
**Pull request by alexander-aleksandrov**
This PR contains the changes in wrapping for 100 column right margin.

### CODE REVIEW

**Comment by alexander-aleksandrov** [comment]
@dmytro-grankin PTAL

File: client/src/main/java/org/spine3/base/Commands.java [hunk 1]
```diff
@@ -269,7 +270,8 @@ public int compare(Command o1, Command o2) {
      * Checks if the file is for commands.
      *
      * @param file a descriptor of a {@code .proto} file to check
-     * @return {@code true} if the file name ends with the {@link #FILE_NAME_SUFFIX}, {@code false} otherwise
+     * @return {@code true} if the file name ends with the {@link #FILE_NAME_SUFFIX},
+     *                      {@code false} otherwise
```
Comment (dmytro-grankin) [inline client/src/main/java/org/spine3/base/Commands.java]: Possible, it's nice formatting for a Javadoc, but `idea`'s `reformat code` will reformat a Javadoc with such formatting. We should avoid this here and in similar cases.

File: client/src/main/java/org/spine3/util/Math.java [hunk 2]
```diff
@@ -20,6 +20,8 @@
 
 package org.spine3.util;
 
+import static java.lang.String.*;
```
Comment (dmytro-grankin) [inline client/src/main/java/org/spine3/util/Math.java]: Do not `static` use imports like this, specify a particular import. For example, use `java.lang.String.format`. Do it in all similar cases.

File: client/src/test/java/org/spine3/testdata/TestCommandContextFactory.java [hunk 3]
```diff
@@ -60,7 +60,8 @@ public static CommandContext createCommandContext(UserId userId, CommandId comma
                                                              .setActor(userId)
                                                              .setTimestamp(when)
                                                              .setZoneOffset(UTC)
-                                                             .setTenantId(TenantId.newBuilder().setValue(newUuid()));
+                                                             .setTenantId(TenantId.newBuilder()
+                                                                                  .setValue(newUuid()));
```
Comment (dmytro-grankin) [inline client/src/test/java/org/spine3/testdata/TestCommandContextFactory.java]: This row is larger than 100 symbols.

File: server/src/main/java/org/spine3/server/storage/BulkStorageOperationsMixin.java [hunk 4]
```diff
@@ -40,12 +41,13 @@
      *
      * <p>The size of {@link Iterable} returned is always the same as the size of given IDs.
      *
-     * <p>In case there is no record for a particular ID, {@code null} will be present in the result instead.
-     * In this way {@code readMultiple()} callees are able to track the absence of a certain element by comparing
-     * the input IDs and resulting {@code Iterable}.
+     * <p>In case there is no record for a particular ID, {@code null} will be present
+     * in the result instead. In this way {@code readMultiple()} calls are able to track
```
Comment (dmytro-grankin) [inline server/src/main/java/org/spine3/server/storage/BulkStorageOperationsMixin.java]: Is really `callees` should be replaced by `calls`?

Comment (alexander-aleksandrov) [inline server/src/main/java/org/spine3/server/storage/BulkStorageOperationsMixin.java]: No, my mistake fixed

File: client/src/main/java/org/spine3/base/Commands.java (hunk 1 above)
Comment (alexander-aleksandrov) [inline client/src/main/java/org/spine3/base/Commands.java]: Done

File: client/src/test/java/org/spine3/testdata/TestCommandContextFactory.java (hunk 3 above)
Comment (alexander-aleksandrov) [inline client/src/test/java/org/spine3/testdata/TestCommandContextFactory.java]: Done

**Comment by alexander-aleksandrov** [comment]
@dmytro-grankin PTAL I've added few fixes. Please take a notice that we should not change links that are longer than 100 columns.

File: client/src/test/java/org/spine3/protobuf/MessageFieldShould.java [hunk 5]
```diff
@@ -28,6 +28,7 @@
 import static com.google.protobuf.Descriptors.FieldDescriptor.JavaType;
 import static org.junit.Assert.assertEquals;
 import static org.spine3.base.Identifiers.newUuid;
+import static org.spine3.protobuf.MessageField.*;
```
Comment (dmytro-grankin) [inline client/src/test/java/org/spine3/protobuf/MessageFieldShould.java]: Optimize imports for this file, then `...MessageField.*` import will disappear.

**Pull request merged**